pvlib/tests/test_spectrum.py

```diff
@@ -315,3 +315,44 @@ def test_spectral_factor_caballero_supplied_ambiguous():
     with pytest.raises(ValueError):
         spectrum.spectral_factor_caballero(1, 1, 1, module_type=None,
                                            coefficients=None)
+
+
+@pytest.mark.parametrize("module_type,expected", [
+    ('asi', np.array([1.15534029, 1.1123772, 1.08286684])),
+    ('fs-2', np.array([1.0694323, 1.04948777, 1.03556288])),
+    ('fs-4', np.array([1.05234725, 1.037771, 1.0275516])),
+    ('multisi', np.array([1.03310403, 1.02391703, 1.01744833])),
+    ('monosi', np.array([1.03225083, 1.02335353, 1.01708734])),
+    ('cigs', np.array([1.01475834, 1.01143927, 1.00909094])),
+])
+def test_spectral_factor_pelland(module_type, expected):
+    ams = np.array([1.0, 1.5, 2.0])
+    kcs = np.array([0.4, 0.6, 0.8])
+    out = spectrum.spectral_factor_pelland(ams, kcs,
+                                           module_type=module_type)
+    assert np.allclose(expected, out, atol=1e-3)
```

Comment:
Seems like we can do better than `1e-3` here.  Looking at the expected values, how about `1e-7` or `1e-8`?